lib/models/customer_info_wrapper.dart

```diff
@@ -43,12 +43,12 @@ class CustomerInfo with _$CustomerInfo {
     @JsonKey(name: 'requestDate') String requestDate, {
 
     /// The latest expiration date of all purchased skus
-    @JsonKey(name: 'latestExpirationDate', nullable: true)
+    @JsonKey(name: 'latestExpirationDate')
```

Comment:
`nullable: true` is deprecated and doesn't have any effect anymore

Reply:
This wasn't failing before, so I updated the dependency on `json_annotation` to 4.8.1